### PR TITLE
[MUI-380]: Update design of banner message

### PR DESCRIPTION
**Pull request by mr-teddy-bear**
# Description

Code review for banner message component

## Jira ticket(optional)

[MUI-380](https://minterestfinance.atlassian.net/jira/software/projects/MUI/boards/7?assignee=62591503eee0a9006ab98b74&selectedIssue=MUI-380)

## Type of change

Please pick options that are relevant.

- [ ] Bug fix OR refactoring (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature or refactoring that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/components/HeaderMessage/HeaderMessage.tsx [hunk 1]
```diff
@@ -13,73 +13,61 @@ export type HeaderMessagePropsType = {
...
-height: fit-content;
-background: #f6f7f9;
-display: flex;
-align-items: center;
-justify-content: center;
-padding: 16px 120px 18px 94px;
+const Body = styled('div')(({ theme }) => ({
+  width: '100%',
+  height: 'fit-content',
+  background: theme.palette.custom.light,
```
Comment (gavrilin-remote) [inline src/components/HeaderMessage/HeaderMessage.tsx:19]: not sure we should use custom here, but ok

Comment (gavrilin-remote) [reply src/components/HeaderMessage/HeaderMessage.tsx:19]: I will replace it later

File: src/components/HeaderMessage/HeaderMessage.tsx [hunk 2]
```diff
@@ -13,73 +13,61 @@ export type HeaderMessagePropsType = {
...
-margin-top: 8px;
-font-family: 'Open Sans';
-font-style: normal;
-font-weight: 400;
-font-size: 16px;
-line-height: 150%;
-color: ${theme.palette.action.disabled};
-`
-);
+const Description = styled('div')(({ theme }) => ({
```
Comment (gavrilin-remote) [inline src/components/HeaderMessage/HeaderMessage.tsx:62]: replace div with Typography

Comment (gavrilin-remote) [reply src/components/HeaderMessage/HeaderMessage.tsx:62]: and use proper variant, we already have such font
`fontFamily: 'Open Sans',
  fontStyle: 'normal',
  fontWeight: 400,`

**Review by gavrilin-remote — CHANGES_REQUESTED** [review]

File: src/components/HeaderMessage/HeaderMessage.tsx (hunk 2 above)
Comment (mr-teddy-bear) [reply src/components/HeaderMessage/HeaderMessage.tsx]: Done, but something wrong after merging with main -_-
Linter hate me)

**Review by gavrilin-remote — APPROVED** [review]

**Review by BerRossDolmat — APPROVED** [review]

**Review by marinakulsha — APPROVED** [review]

File: src/components/HeaderMessage/HeaderMessage.tsx [hunk 3]
```diff
@@ -13,73 +13,56 @@ export type HeaderMessagePropsType = {
...
-  flex-wrap: wrap;
-}
+const Title = styled(Typography)(({ theme }) => ({
+  color: theme.palette.custom.dark,
+}));
 
-`
-);
+const TextWrapper = styled('div')(() => ({
+  maxWidth: '729px',
```
Comment (BerRossDolmat) [inline src/components/HeaderMessage/HeaderMessage.tsx:59]: magic number for width, not cool in ui kit that can be used in different layout

File: src/theme.ts [hunk 4]
```diff
@@ -31,12 +31,14 @@ export const getTheme = (mode: PaletteMode): Theme =>
       secondary: {
         main: '#061953',
       },
-      info:{
-        main: '#fafafa'
+      info: {
+        main: '#fafafa',
       },
       // example todo remove later
```
Comment (BerRossDolmat) [inline src/theme.ts:37]: needs to be removed?

File: src/components/HeaderMessage/HeaderMessage.tsx [hunk 5]
```diff
@@ -13,73 +13,56 @@ export type HeaderMessagePropsType = {
...
+  [theme.breakpoints.down('sm')]: {
+    padding: '8px 40px 8px 20px',
+  },
+}));
 
-`
-);
+const Message = styled('div')(({ theme }) => ({
+  position: 'relative',
+  width: '1226px',
```
Comment (BerRossDolmat) [inline src/components/HeaderMessage/HeaderMessage.tsx:36]: magic number in width

**Pull request merged**